include/functions.php

```diff
@@ -1,55 +1,67 @@
 <?php
-    function getPosts(){
-        $Post = array(
-            0=>array(
-                "PostId"=>0,
-                "Title"=>"Hi, I'm Kay!",
-                "Body"=>"<h2>What is this place?
-                <p>Kay is the creator of this website, an online community of people who are on a journey towards understanding, appreciation, and unity of ourselves.
-                Here we are working to improve ourselves which will end in countless improvements in every aspect of our lives. (purpose)
+function getPosts(){
+    $Post = array(
```

Comment:
This is very minor, but I just want to make sure you're aware of it. You indentations are using four spaces rather than one tab. This is most likely a setting in your code editor that makes it so when you hit the "tab" key, it inserts four spaces.

This is not necessarily a bad thing. Many programmers prefer spaces over tabs. I personally prefer tabs over spaces (I'm happy to explain my reasoning to you if you care, but you probably don't). Just want to make sure you're aware of it so that whatever you do, you're doing it deliberately. If you prefer tabs, there's a setting in your code editor that controls this.

If you're interested in some nerdy humor, here's a scene from Silicon Valley about this debate: https://www.youtube.com/watch?v=SsoOG6ZeyUI